gapii/cc/vulkan_mid_execution.cpp

```diff
@@ -446,8 +460,9 @@ void VulkanSpy::serializeGPUBuffers(StateSerializer* serializer) {
         memory::Observation observation;
         observation.set_pool(deviceMemory->mData.pool_id());
         observation.set_base(bind.mmemoryOffset + offset);
-        serializer->sendData(&observation, true, stage.GetMappedMemory(),
+        serializer->sendData(&observation, true, stage->GetMappedMemory(),
                              chunkSize);
+        stage->UnmapMemory();
```

Comment:
You shouldn't have to unmap this buffer.
Leaving it mapped the entire time is probably simpler.
Just have to flush it before reading it.

Reply:
done